packages/claw/package.py

```diff
@@ -23,6 +23,8 @@ class Claw(CMakePackage):
     version('1.2.1', commit='939989ab52edb5c292476e729608725654d0a59a', submodules=True)
     version('1.2.0', commit='fc9c50fe02be97b910ff9c7015064f89be88a3a2', submodules=True)
     version('1.1.0', commit='16b165a443b11b025a77cad830b1280b8c9bcf01', submodules=True)
+    
+    variant('omni-master', default=False, description='Build with the master version of the omni-compiler')
```

Comment:
actually even better would be able to specify a branch+fork so that I could use the source of the PR

Reply:
we could set to a value for master, the only thing which will not work is to change the fork. Probably need some change in claw